Kustomize installation test failing

### What happened?

The kustomize test have been failing for some time.

![Capture d’écran du 2023-11-24 09-13-31](https://github.com/apache/camel-k/assets/6067789/f65ebae2-89d2-4556-8d31-57f9b2df9c2a)


### Steps to reproduce

_No response_

### Relevant log output

```shell
The logs contains:

  No IntegrationPlatform resource in default namespace
  Camel K Integration Platform removed from namespace default
  Camel K Config Maps removed from namespace default
  Camel K Registry Secret removed from namespace default
  Camel K Platform Kamelets removed from namespace default
  Camel K Platform Camel Catalogs removed from namespace default
  Camel K Operator removed from namespace default
  Camel K Role Bindings removed from namespace default
  Camel K Role Bindings removed from namespace kube-public
  Camel K Roles removed from namespace default
  Camel K Service Accounts removed from namespace default
  Camel K Custom Resource Definitions removed from cluster
  Camel K Cluster Role Bindings removed from cluster
  Camel K Cluster Roles removed from cluster
  Using alternative make directory on path: /tmp/camel-k-105473497/install
      operator_test.go:51: 
          No future change is possible.  Bailing out early after 0.000s.
          Expected
              <int>: 2
          to match exit code:
              <int>: 0
      test_util.go:67: Output from exec command:
          make[1]: Entering directory '/tmp/camel-k-105473497/install'
          make[1]: Leaving directory '/tmp/camel-k-105473497/install'
          
      test_util.go:68: Error from exec command:
          Makefile:181: *** *** CRD API FAILURE: ERROR: CRD API version 'v1' is not supported. ****.  Stop.
          
      dump.go:48: -------------------- start dumping namespace test-b1622017-46be-4dba-ba71-d3438fb4d9eb --------------------
      test_support.go:2742: Error while dumping namespace test-b1622017-46be-4dba-ba71-d3438fb4d9eb: the server could not find the requested resource (get integrations.camel.apache.org)
  Unable to retrieve the operator version: no matches for kind "IntegrationPlatform" in version "camel.apache.org/v1"
  Error: the server could not find the requested resource (get integrationplatforms.camel.apache.org)
  Camel K cluster setup completed successfully
```


### Camel K version

main branch

--- a/script/check_crd_api_support.sh
+++ b/script/check_crd_api_support.sh
@@ -53,9 +53,9 @@ fi
 crd_version=$("${client}" explain customresourcedefinitions | grep VERSION | awk '{print $2}')
 api="apiextensions.k8s.io"
 
-if [ "${crd_version}" == "${api}/v1beta1" ]; then
+if [ "${crd_version}" == "${api}/v1beta1" ] || [ "${crd_version}" == "v1beta1" ]; then
 	echo "ERROR: CRD API version is too old to install camel-k in this way. Try using the client CLI app, which is able to convert the APIs."
-elif [ "${crd_version}" != "${api}/v1" ]; then
+elif [ "${crd_version}" != "${api}/v1" ] && [ "${crd_version}" != "v1" ]; then
 	echo "ERROR: CRD API version '${crd_version}' is not supported."
 else
 	echo "OK"

